#! /bin/sh -e
## 20_usr_bin.dpatch by <luther@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use the correct /usr/bin interpreters for:
## DP:  - hello.tcl (wish intepreter)
## DP:  - ocamlsize (perl interpreters)

if [ $# -ne 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi
case "$1" in
    -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
    -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0
@DPATCH@
--- ocaml-3.07-3.06.99.2003.07.05.orig/otherlibs/labltk/examples_labltk/hello.tcl
+++ ocaml-3.07-3.06.99.2003.07.05/otherlibs/labltk/examples_labltk/hello.tcl
@@ -1,4 +1,4 @@
-#!/usr/local/bin/wish4.0
+#!/usr/bin/wish
 
 button .hello -text "Hello, TclTk!"
 
--- ocaml-3.07-3.06.99.2003.07.05.orig/tools/ocamlsize
+++ ocaml-3.07-3.06.99.2003.07.05/tools/ocamlsize
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 foreach $f (@ARGV) {
     open(FILE, $f) || die("Cannot open $f");
